api/swagger-spec/apps_v1beta1.json

```diff
@@ -6010,6 +6047,29 @@
       "type": "integer",
       "format": "int32",
       "description": "replicas is the number of actual replicas."
+     },
+     "readyReplicas": {
+      "type": "integer",
+      "format": "int32",
+      "description": "readyReplicas is the number of replicas with a Ready Condition."
+     },
+     "currentReplicas": {
+      "type": "integer",
+      "format": "int32",
+      "description": "currentReplicas is the number of replicas at the CurrentRevision"
+     },
+     "updatedReplicas": {
+      "type": "integer",
+      "format": "int32",
+      "description": "updatedReplicas is the number of replicas"
+     },
+     "currentRevision": {
+      "type": "string",
+      "description": "currentRevision is the current revision of the StatefulSet"
+     },
+     "updateRevision": {
+      "type": "string",
+      "description": "updateRevision is the revision the StatefulSet is attempting to roll out"
```

Comment:
nit: comma after `revision` as in `updateRevision is the revision, the StatefulSet is ...`